automatic/git.portable/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,32 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$is64bit = Get-ProcessorBits 64
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$filePath32 = "$toolsDir\PortableGit-2.11.0-32-bit.7z.exe"
+$filePath64 = "$toolsDir\PortableGit-2.11.0-64-bit.7z.exe"
+$filePath = if ($is64bit -and $env:chocolateyForceX86 -eq $false) { $filePath64 } else { $filePath32 }
+$installLocation = Join-Path $(Get-ToolsLocation) 'git'
+
+Get-ChocolateyUnzip -FileFullPath $filePath -Destination $installLocation
+
+# Add to path
+$gitBinPath = Join-Path $installLocation 'bin'
+Install-ChocolateyPath $gitBinPath 'Machine'
+$env:Path = "$($env:Path);$gitBinPath"
+
+# Generate ignore files
```

Comment:
```powershell
# Generate ignore files
Write-Host "Generate ignore files"
$files = get-childitem $installLocation -include *.exe -recurse
foreach ($file in $files) {
  if (!($file.FullName -match '\\cmd\\git.exe$|\\git-bash.exe$|ssh[^\\]$')) {
    New-Item "$file.ignore" -type file -force | Out-Null
  }
}
```

This is not needed, its required when in chocolatey lib folder. You can remove this.